pkg/reconciler/cluster/controller.go

```diff
@@ -206,20 +270,48 @@ func (c *Controller) deletedCluster(obj interface{}) {
 }
 
 func RegisterClusterCRD(cfg *rest.Config) error {
```

Comment:
Maybe rename to `RegisterCRDs` with a doc comment about what it's going to register?

Reply:
Fixed in commit 6a4dcd07a4a268ef7dd0158950e2a1c8452b9eb3